modules/custom/az_event/az_event_trellis/az_event_trellis.links.task.yml

```diff
@@ -6,7 +6,7 @@ az_event_trellis.az_event_trellis_settings_task:
 
 az_event.import_form_task:
   title: 'Trellis event importer'
-  route_name: az_event_trellis.az_event_trellis_import_form
+  route_name: view.az_event_trellis_import.page_1
```

Comment:
Should we also remove the old form class?

Reply:
I have removed the original form class.